### DISCUSSION TITLE
Embedding a Vimeo video

### DISCUSSION THREAD

**Discussion by scvn1** [awcodes/filament-tiptap-editor#122] [Q&A]
Is it possible to embed Vimeo video code in the content? I don't mean as an example of code, but real player code that will play the video.

**Comment by awcodes** [comment]
There is an oembed tool which you can use to paste the Vimeo url into that will get you what you're looking for.

**Reply by scvn1** [reply]
After some tinkering with the URL format, I discovered it only works with this specific format of Vimeo video URLs (for anyone else that reads this):
https://vimeo.com/{video_id}

If you paste in a URL like this, it just fails silently with no errors explaining what is going on, so I thought Vimeo wasn't supported at first:
https://player.vimeo.com/video/{video_id}

**Reply by awcodes** [reply]
Hmm. It should be compensating that. Can you report it as an issue and I'll look into it.

**Reply by awcodes** [reply]
This will be fixed with the next release. Still no curator integration for videos yet. But this update will lay the groundwork for it by supporting native video embeds via a url.

**Reply by scvn1** [reply]
Great! Do you still need a bug report filed or all good?

**Reply by awcodes** [reply]
No need. Thank you.

**Comment by awcodes** [comment]
This has been released.

**Reply by scvn1** [reply]
That fixed several things and the UI is better, too. Love the floating widget as well. Thanks for this!
